### ISSUE TITLE
ionic-pwa failing to start

### ISSUE THREAD

**Issue by adrm** [ionic-team/stencil#3013]
**Stencil version:**

```
 @stencil/core@2.7.0
```

**I'm submitting a:**

[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://stencil-worldwide.herokuapp.com/ or https://forum.ionicframework.com/

**Current behavior:**

A new stencil 2.7 project will fail when trying to run npm start. The problem disappears if the version is changed back to 2.6. The resulting debug log is the following.

```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'start' ]
2 info using npm@6.14.13
3 info using node@v14.17.0
4 verbose run-script [ 'prestart', 'start', 'poststart' ]
5 info lifecycle testproject@0.0.1~prestart: testproject@0.0.1
6 info lifecycle testproject@0.0.1~start: testproject@0.0.1
7 verbose lifecycle testproject@0.0.1~start: unsafe-perm in lifecycle true
8 verbose lifecycle testproject@0.0.1~start: PATH: /usr/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/var/home/adrian/Proyectos/testproject/node_modules/.bin:/var/home/adrian/.deno/bin:/var/home/adrian/.cargo/bin:/var/home/adrian/.local/bin:/var/home/adrian/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
9 verbose lifecycle testproject@0.0.1~start: CWD: /var/home/adrian/Proyectos/testproject
10 silly lifecycle testproject@0.0.1~start: Args: [ '-c', 'stencil build --dev --watch --serve' ]
11 silly lifecycle testproject@0.0.1~start: Returned: code: 1  signal: null
12 info lifecycle testproject@0.0.1~start: Failed to exec start script
13 verbose stack Error: testproject@0.0.1 start: `stencil build --dev --watch --serve`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:376:20)
13 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:376:20)
13 verbose stack     at maybeClose (internal/child_process.js:1055:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5)
14 verbose pkgid testproject@0.0.1
15 verbose cwd /var/home/adrian/Proyectos/testproject
16 verbose Linux 5.13.12-200.fc34.x86_64
17 verbose argv "/usr/bin/node" "/usr/bin/npm" "start"
18 verbose node v14.17.0
19 verbose npm  v6.14.13
20 error code ELIFECYCLE
21 error errno 1
22 error testproject@0.0.1 start: `stencil build --dev --watch --serve`
22 error Exit status 1
23 error Failed at the testproject@0.0.1 start script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

And the console returned this error:

```
[24:48.9]  @stencil/core
[24:49.0]  v2.7.0 🌟

[ ERROR ]  uncaught cli error: TypeError: Cannot read property 'match' of undefined
```

Updating to node v16 gave the same error.

**Expected behavior:**

A newly created app should start without issue.

**Steps to reproduce:**

Create an app with `npm init stencil` and type `ionic-pwa`. Follow recommended steps: `cd` into it, `npm install` and `npm start`.

**Other information:**

**Comment by rwaskiewicz** [comment]
👋 Thanks for the report!

Unfortunately, I'm unable to reproduce this on either a Linux machine or in MacOS using the versions of npm & node that are listed in the debug log.

To help further diagnose this issue, can you do two things for me?

1. Create a public repository with your output of running `npm init stencil` and link it here? That should help us ensure that something weird didn't happen in the initialization steps
2. From the project's directory, run `npx stencil telemetry off` and try running `npm start` again?

**Comment by adrm** [comment]
Here is the public repository with the output: https://github.com/adrm/stencil-issue-3013

I just did `npm init stencil`, and then `cd` into the repo, `npm install` and `npm start` to check that the error happened again, and it did.

When trying to run the telemetry command, I received the same `uncaught cli error: TypeError: Cannot read property 'match' of undefined` error.

**Comment by rwaskiewicz** [comment]
Thanks! I think I have an idea of what’s going on here - I’ll dive into the issue further tomorrow once the work week starts

**Comment by rwaskiewicz** [comment]
@adrm this should be fixed in v2.7.1 of Stencil. Thanks again!

**Comment by adrm** [comment]
Thanks to you! Keep up the great work.
